README.md

```diff
@@ -2,93 +2,161 @@
+
+If you said `if petName != nil`, give yourself a pat on the back, because you're right! You can check to see if a value in an `Optional` exists by testing if it is equal to `nil` or not. Try this if statement in your playground:
 
 ```swift
-if myPet != nil {
-    print("I have a pet \(myPet)")
+if petName != nil {
+    print("My friend's pet is \(petName)")
 } else {
-    print("We have no pet :(")
+    print("My friend doesn't have a pet 😞")
 }
 
-// prints "I have a pet Optional("Turtle 🐢")"
+// prints "My friend's pet is Optional("Scooter 🐢")"
 ```
 
-* Well, it's still printing that optional bit. Here is where we could introduce forced unwrapping.
-* Here we can introduce *optional binding* which we use to find out whether an optional contains a value, and if so, to make that value available as a temporary constant or variable.
-* I've seen students get confused with optional binding thinking the temporary constant made has to be the same name of the optional, so I think it's a good idea to show them something like the following:
+Voilà! You checked to see if `petName != nil`, and since it doesn't, the first branch of your if statement was executed. Try going back and removing or commenting out the second line of your playground (`petName = "Scooter 🐢"`) and see what is printed instead.
+
+But wait. We said that unwrapping an `Optional` wouldn't print that "Optional(...)" bit in the console, but it's still being printed! That's because you haven't actually _unwrapped_ the value yet—you've only checked to see if it's `nil` or not.
+
+To unwrap an `Optional`, you use a Swift feature called _optional binding_. In optional binding, you check to see if a value exists (is not `nil`) _and_ set it equal to a constant. That constant is now available for use within the first branch of the if statement. More important, it is bound to the actual _value_ wrapped by the `Optional`.
+
+Try this in your playground:
 
 ```swift
-if let myPet = myPet {
-    print("Scooter is the name of my pet \(myPet)")
-    // prints "Scooter is the name of my pet Turtle 🐢"
+if let petName = petName {
+    print("My friend's pet is \(petName)")
 }
 
-if let scooterTheTurtle = myPet {
-    print("My pet \(scooterTheTurtle) is pretty cool, he loves to eat lettuce.")
-    // prints "My pet Turtle 🐢 is pretty cool, he loves to eat lettuce."
+// prints "My friend's pet is Scooter 🐢"
+```
+
+What do you see in the console now?
+
+You should see "My friend's pet is Scooter 🐢". Look—no more "Optional(...)" stuff!
+
+Let's break down at if statement a bit. Within the condition of the if statement, you wrote `let petName = petName`. That looks like a constant definition, doesn't it? Well, it is. What that line says is this: "If `petName` has a value, unwrap it and assign it to the name `petName` within the body of this if statement." You can then use `petName` to refer to the _unwrapped_ value (but only within the if statement).
```

Comment:
I was about to suggest you do something like this referring to the 

```swift
let petName = petName
```

but you just did it. Nice